qiskit/pulse/builder.py

```diff
@@ -74,8 +74,8 @@
 
 The builder initializes a :class:`.pulse.Schedule`, ``pulse_prog``
 and then begins to construct the program within the context. The output pulse
-schedule will survive after the context is exited and can be transpiled and executed like a
-normal Qiskit schedule using ``backend.run(transpile(pulse_prog, backend))``.
+schedule will survive after the context is exited and can be executed like a
+normal Qiskit schedule using ``backend.run(pulse_prog)``.
```

Comment:
The providers have not supported executing pulse schedules for a while now. They only support attaching schedules to circuits as calibrations. Since pulse gates are not talked about in this section, that might be too much to reference right here. Maybe just this?

```suggestion
schedule will survive after the context is exited and can be used like a
normal Qiskit schedule.
```